docs/pages/installation/gke.md

```diff
@@ -9,56 +9,94 @@ list: include
+
+### Preflight: Cluster Connectivity
+
+1. Verfiy you connection to an Google Kubernetes Engine Cluster using gCloud CLI.
+1. Login to GCP account using [gcloud auth login](https://cloud.google.com/sdk/gcloud/reference/auth/login).
+1. After a successful login, set the Project Id:
+{% capture code_content %}gcloud config set project [PROJECT_ID]
+{% endcapture %}
+{% include code.html code=code_content %}
+1. After setting the Project Id, set the cluster context.
+{% capture code_content %}gcloud container clusters get-credentials [CLUSTER_NAME] --zone [CLUSTER_ZONE] {% endcapture %}
+{% include code.html code=code_content %}
+1. Verify your kubeconfig's current context.
+{% capture code_content %}kubectl cluster-info{% endcapture %}
+{% include code.html code=code_content %}
+
+### Preflight: Plan your access to Meshery UI
+
+1. If you are using port-forwarding, please refer to the [port-forwarding]({{ site.baseurl }}/reference/mesheryctl/system/dashboard) guide for detailed instructions.
+2. If you are using a LoadBalancer, please refer to the [LoadBalancer]({{ site.baseurl }}/installation/platforms/kubernetes#exposing-meshery-serviceloadbalancer) guide for detailed instructions.
+3. Customize your Meshery Provider Callback URL. Meshery Server supports customizing authentication flow callback URL, which can be configured in the following way:
+
+{% capture code_content %}$ MESHERY_SERVER_CALLBACK_URL=https://custom-host mesheryctl system start{% endcapture %}
+{% include code.html code=code_content %}
+
+Meshery should now be running in your GKE cluster and Meshery UI should be accessible at the `EXTERNAL IP` of `meshery` service.
+
+## Installation: Using `mesheryctl`
+
+Use Meshery's CLI to streamline your connection to your GKE cluster. Configure Meshery to connect to your GKE cluster by executing:
+
+{% capture code_content %}$ mesheryctl system config gke{% endcapture %}
+{% include code.html code=code_content %}
+
+Once configured, execute the following command to start Meshery.
+
+{% capture code_content %}$ mesheryctl system start{% endcapture %}
+{% include code.html code=code_content %}
+
+If you encounter any authentication issues, you can use `mesheryctl system login`. For more information, click [here](/guides/mesheryctl/authenticate-with-meshery-via-cli) to learn more.
+
+## Installation: Using Helm
+
+For detailed instructions on installing Meshery using Helm V3, please refer to the [Helm Installation](/installation/helm) guide.
+
+# Post-Installation Steps
+
+## Access Meshery UI
+
+To access Meshery's UI, please refer to the [instruction](/tasks/accessing-meshery-ui) for detailed guidance.
+
+Note:   _You can also verify health of your system using [mesheryctl system check](/reference/mesheryctl/system/check)_
+
+# Suggested Reading
+
+<!-- {% include suggested-reading.html diffName="true" isDiffTag="true" diffTag=tag %} -->
+{% include related-discussions.html tag="mesheryctl" %}
```

Comment:
```suggestion
{% include suggested-reading.html language="en" %}

{% include related-discussions.html tag="meshery" %}
```

The logic in `suggested-reading.html` was _just_ updated, which means that it is now (hopefully) useful.

Reply:
yeah it is useful.

Reply:
![image](https://github.com/meshery/meshery/assets/144536269/ea48180f-d902-4d19-bbe4-5052a21f4b49)